neo/Consensus/ConsensusService.cs

```diff
@@ -393,7 +393,7 @@ private void CheckPayloadSignatures()
                 Block block = context.MakeHeader();
                 if (block == null) return;
                 context.State |= ConsensusState.CommitSent;
-                context.FinalSignatures[context.MyIndex] = block.Sign(context.KeyPair);
+                context.FinalSignatures[context.MyIndex] = context.SignBlock(block);
                 //SignAndRelay(context.MakeCommitAgreement(context.FinalSignatures[context.MyIndex]));
                 system.LocalNode.Tell(new LocalNode.SendDirectly { Inventory = context.MakeCommitAgreement(context.FinalSignatures[context.MyIndex]) });
```

Comment:
@igormcoelho, maybe one signature is missing here on MakeCommitAgreement.
Where is this payload being signed now?

Reply:
I see, automatically in MakeSignedPayload, right?